tests/dune

```
@@ -1,5 +1,5 @@
 (test
   (name tests)
-  (deps sample.opam)
+  (deps sample-opam)
   (package opam-file-format)
   (libraries alcotest opam-file-format))
```

```suggestion
  (libraries alcotest opam-file-format))

(rule (copy sample-opam sample.opam))
```
and all the other changes can be removed.

---

done